cmd/pg/backup_fetch.go

```diff
@@ -16,18 +19,26 @@ For information about pattern syntax view: https://golang.org/pkg/path/filepath/
 	restoreSpecDescription        = "Path to file containing tablespace restore specification"
 	reverseDeltaUnpackDescription = "Unpack delta backups in reverse order (beta feature)"
 	skipRedundantTarsDescription  = "Skip tars with no useful data (requires reverse delta unpack)"
+	targetUserDataDescription     = "Fetch storage backup which has the specified UserData"
```

Comment:
UserData looks like a label. Let's rename it.

Reply:
fixed in 
9874d81

Reply:
Probably you didn't get it. "UserData/user_data -> label" everywhere